sufsara_tutorial.Rmd

```diff
@@ -1,28 +1,54 @@
+Once you have been granted access to the *Wageningen University Geoscripting* collaboration *and* you have set up 2FA, you should be able to access your VM. Navigate back to the [starting page](https://portal.live.surfresearchcloud.nl/), and scroll down the page to the 'Workspaces' section. You should see multiple available workspaces (VMs). Please only make use of the workspace with your group name/number. If a workspace with your group name/number does not exist, please contact the staff.
 
-## Start a VM
+At the right of every workspace, there should be a (yellow) button that allows access to the workspace. Press that button for the workspace you want to access. A login-page will show up. Enter the following information:
 
-<!--### "Cloud" view-->
-You can start a virtual machine by going to the *VMs* tab and clicking the + button. Select the "Ubuntu 16.04 GeoScripting" template and press "Create" (do not change the size of the disks). Wait a bit (several minutes) until your VM enters the ready state (icon turns green). Note that the page does not update automatically: you need to click the *Refresh* (circling arrows) button in the SURFsara UI (**not** in your browser!) to update the view.
+* Use the standard Session type (Xorg)
+* Your username is your SURF researchcloud username, which is not the same as your WUR username (!). You can find your SURF username in your [profile page](https://portal.live.surfresearchcloud.nl/profile)
+* Your password is the 2FA code
 
-<!--### "User" view
-Next, press "Change view" and set it to "user". From the Dashboard, go to Virtual Resources → Templates, select the "Ubuntu 16.04 GeoScripting" template and press "Clone". Give it a name you like: this will become your very own virtual machine. A *Template* is a set of settings for your virtual machine, including the number of cores, memory and hard disks.
+If your VM is coming out of 'sleep mode', and you have to fill in a password to unlock it, you should (again) make use of your 2FA code. Happy coding!
 
-You can run it already by clicking on it and pressing "Instantiate", then confirming the settings by pressing "Instantiate" again. You will see your virtual machine running in the Virtual Resources → Virtual Machines page. -->
+# Optional: accessing the VM from your own machine
+
+The tutorial above allows you to access the VM in-browser. There are however alternatives to access the VM directly from your own machine, for which we provide a tutorial below.
+
+## Enrol your SSH key
+
+Navigate to your [SRAM profile page](https://sram.surf.nl/profile). Generate an SSH key on your own machine. Paste this SSH key into the text box, and click the "Save" button. Please refer to online tutorials to find/create an SSH key for your machine, like [this one](https://www.howtogeek.com/762863/how-to-generate-ssh-keys-in-windows-10-and-windows-11/) for Windows.
```

Comment:
The SSH key *generation* is actually also explained in Day 1, it can be easily done using Git GUI too.

Reply:
Also, maybe good to clarify that this is the *public* SSH key, the private key should never be pasted anywhere (and will not be accepted if you try).

Reply:
Well, true that you don't technically need to have Git GUI installed to get to the VMs, but given that Git is a central concept in the course and that Git GUI is really easy to generate keys in, it's a good idea anyway. I think it's easier to just have everyone generate keys the same way throughout the course, across all platforms, than try to deal with everyone trying to use their own method of generating keys.

And yes, Remote Desktop does not need an SSS key, it uses the 2FA. You need either one or the other, not both :)

Reply:
Yes, the SSH key is needed for the command line option, which is actually the most efficient.